soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
Ok, how about that. I disable the ball only when we try to plan straight into it. So the ball isn't moving the anyways. This prevents the messed up constant pathplanning when we try to plan into the ball obstacle while still going around the ball when we need to catch it.